statefun-e2e-tests/statefun-smoke-e2e-java/src/main/java/org/apache/flink/statefun/e2e/smoke/CommandInterpreterAppServer.java

```diff
@@ -30,7 +30,7 @@ public static void main(String[] args) {
     final int numInstances = Integer.parseInt(args[0]);
     final CommandInterpreter interpreter = new CommandInterpreter(new Ids(numInstances));
     final StatefulFunctionSpec FN_SPEC =
-        StatefulFunctionSpec.builder(CommandInterpreterFn.TYPENAME)
+        StatefulFunctionSpec.builder(Constants.CMD_INTERPRETER_FN)
```

Comment:
Shall we follow the pattern like this:
```
import static org.apache.flink.statefun.e2e.smoke.Constants.*;
```
And just do:
```
StatefulFunctionSpec.builder(CMD_INTERPRETER_FN)
```

Reply:
Ok, will do that.